src/cart/utils/docker/Dockerfile.sles.12.3

```diff
@@ -70,6 +70,9 @@ RUN pip install -U setuptools
 RUN pip install -U wheel
 RUN pip install scons==3.0.1; pip3 install -U "meson<0.45.0"
 
+# For CaRT test wrappers
+RUN pip install tabulate; pip3 install tabulate
```

Comment:
We don't support SLES 12.3 any more so this change can be discarded.

Reply:
SLES 12.3 changes removed.  0b0b1b21b29670a2ff42413820a2893cbccea0ac